libr/core/cmd_anal.c

```diff
@@ -10780,6 +10780,14 @@ static int cmd_anal_all(RCore *core, const char *input) {
 			r_core_anal_all (core);
 			r_print_rowlog_done (core->print, oldstr);
 			r_core_task_yield (&core->tasks);
+			
+			// Run afvn in all fcns 
```

Comment:
trailing whitespace

Reply:
will be good if github could show such artifacts in red background or sthg